src/main/scala/viper/gobra/translator/util/ViperWriter.scala

```diff
@@ -370,6 +370,42 @@ object ViperWriter {
       } yield ()
     }
 
+    /* Emits Viper statements. */
+    def assertWithDefaultReason(cond: vpr.Exp, error: Source.Verifier.Info => VerificationError): Writer[Unit] = {
```

Comment:
`assertWithDefaultReason` and `exhaleWithDefault` are currently unused, aren't they?

Reply:
Yes , they are unused, but I think that they are useful.